x/ccv/provider/keeper/keeper.go

```diff
@@ -1300,3 +1300,175 @@ func (k Keeper) DeleteConsumerCommissionRate(
 	store := ctx.KVStore(k.storeKey)
 	store.Delete(types.ConsumerCommissionRateKey(chainID, providerAddr))
 }
+
+// SetValidatorsPowersCap sets the power-cap value `p` associated to chain with `chainID`
+func (k Keeper) SetValidatorsPowersCap(
```

Comment:
```suggestion
func (k Keeper) SetValidatorsPowerCap(
```
Unify the name, i.e. is it Powers or Power?